kickstarts/partials/post/source_setup.ks.erb

```diff
@@ -9,22 +9,25 @@ ssui_root="$repos_root/manageiq-ui-self_service"
 
 rm -rf $app_root
 
-git clone <%= @appliance_checkout.remote %> $appliance_root
+git clone <%= "--separate-git-dir /tmp/appliance.git" if @build_type == "release" %> \
+        <%= @appliance_checkout.remote %> $appliance_root
```

Comment:
Thanks, I didn't see the hidden comments.

I honestly don't know if people use git on the current release builds to know if killing the .git directory would be a problem.  Maybe @geertj could make an announcement on talk.manageiq.org that future release builds will not be git enabled.  Do we start this with the next darga build or the E release?

I don't really have much need for git enabled release builds but this could affect how we(I) suggest people [apply a bug fix using patch instead of git am](http://talk.manageiq.org/t/vcenter-refresh-issues-in-rc2/650/13).  I don't even know if patch is installed by default.  I would assume so.